typings/context.d.ts

```diff
@@ -4,7 +4,7 @@ import * as tt from './telegram-types'
 
 import { Telegram } from './telegram'
 
-export declare class TelegrafContext {
+export declare class TelegrafContext<StateType = any> {
```

Comment:
👎, this is just implicit casting, `TelegrafContext<T>` is assignable to `TelegrafContext<any>` for any `T`. Do `state: any` for now.

Reply:
```suggestion
export declare class TelegrafContext {
```